src/DynamoCore/Interfaces/IRenderPackageFactory.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Autodesk.DesignScript.Interfaces;
+
+namespace Dynamo.Interfaces
+{
+    public interface IRenderPackageFactory
+    {
+        int MaxTessellationDivisions { get; set; }
+        IRenderPackage CreateRenderPackage();
```

Comment:
Let's make sure we add some basic documentation for this.  What is `IRenderPackageFactory`?  Why would I need it?